apps/aehttp/priv/swagger.json

```diff
@@ -0,0 +1,186 @@
+{
+  "swagger" : "2.0",
+  "info" : {
+    "description" : "This is the [Aeternity](https://www.aeternity.com/) Epoch API.",
+    "version" : "1.0.0",
+    "title" : "Aeternity Epoch",
+    "termsOfService" : "https://www.aeternity.com/terms/",
+    "contact" : {
+      "email" : "apiteam@aeternity.com"
+    }
+  },
+  "basePath" : "/v1",
+  "tags" : [ {
+    "name" : "external",
+    "description" : "External API"
+  } ],
+  "schemes" : [ "http" ],
+  "paths" : {
+    "/ping" : {
+      "get" : {
+        "tags" : [ "external" ],
+        "description" : "",
+        "operationId" : "Ping",
+        "produces" : [ "application/json" ],
+        "parameters" : [ ],
+        "responses" : {
+          "200" : {
+            "description" : "Sample response",
+            "schema" : {
+              "$ref" : "#/definitions/Ping"
+            }
+          }
+        },
+        "security" : [ ]
+      }
+    },
+    "/top" : {
+      "get" : {
+        "tags" : [ "external" ],
+        "description" : "Get the top block header",
+        "operationId" : "GetTop",
+        "consumes" : [ "application/json" ],
+        "produces" : [ "application/json" ],
+        "parameters" : [ ],
+        "responses" : {
+          "200" : {
+            "description" : "successful operation",
+            "schema" : {
+              "$ref" : "#/definitions/Header"
+            }
+          }
+        },
+        "security" : [ ]
+      }
+    },
+    "/block" : {
```

Comment:
What is the difference between GET /top and GET /block?

Reply:
GET /block will have a different structure once the serialization of blocks is ready